deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/api/Model.java

```diff
@@ -53,6 +53,13 @@
      */
     void setListeners(IterationListener... listeners);
 
+    /**
+     * This method ADDS additional IterationListener to existing listeners
+     *
+     * @param listener
+     */
+    void addListener(IterationListener listener);
```

Comment:
Good idea. Minor improvement: we might as well make this ```addListeners(IterationListener...)``` and ```addListeners(Collection<?
 extends IterationListener>)```
That would save users having to chain addListener calls for multiple listeners.